imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml

```
@@ -1,30 +1,42 @@
-# TODO: There are several FILLVALs defined here that do not match standard
-#       convention (see https://imap-processing.readthedocs.io/en/latest/
-#       external-tools/cdf/cdf_requirements.html#fillval
-#       These should be updated.
+# <=== Useful Variables ===>
+int_fillval: &int_fillval -9223372036854775808
+uint8_fillval: &uint8_fillval 255
+uint16_fillval: &uint16_fillval 65535
+uint32_fillval: &uint32_fillval 4294967295
+real_fillval: &real_fillval -1.0e+31
+
+min_int: &min_int -9223372036854775808
+min_epoch: &min_epoch -315575942816000000
+
+max_int: &max_int 9223372036854775807
+max_uint8: &max_uint8 255
+max_uint16: &max_uint16 65535
+max_uint32: &max_uint32 4294967295
+max_epoch: &max_epoch 3155630469184000000
+
 
 # <=== Defaults ===>
 default_attrs: &default
     DEPEND_0: epoch
     DISPLAY_TYPE: no_plot
     FIELDNAM: " "
-    FILLVAL: -9223372036854775808
+    FILLVAL: *uint8_fillval
```

The fillval should align with the min/max datatype you've defined above, so i think you probably want to change this to int_fillval. Alternatively, you could change the min/max to be uint8 values.
```suggestion
    FILLVAL: *int_fillval
```